plenum/common/ledger_manager.py

```diff
@@ -583,6 +583,14 @@ def _add_txn(self, ledgerId, ledger: Ledger, ledgerInfo, txn):
         merkleInfo = ledger.add(self._transform(txn))
         txn[F.seqNo.name] = merkleInfo[F.seqNo.name]
         ledgerInfo.postTxnAddedToLedgerClbk(ledgerId, txn)
+        if self.ownedByNode:
+            node = self.owner
+            req_handler = node.get_req_handler(ledgerId)
+            # Assume, that state updating was called before
+            # state
+            if req_handler and ledgerId == DOMAIN_LEDGER_ID:
```

Comment:
Why do we do it here and not from `postTxnFromCatchupAddedToLedger`? Shouldn't it be called automatically when we update the state?
Looks like we also call it from ledger_req_handler's commit()...